authentik/lib/kerberos/exceptions.py

```diff
@@ -0,0 +1,5 @@
+"""Kerberos exceptions"""
```

Comment:
I think it makes more sense to have these kerberos files in `providers/kerberos`, `/lib` should ideally only contain general helpers (even though the name isn't really all that clear)

Reply:
Done in https://github.com/goauthentik/authentik/pull/6391/commits/2186f717236d6ebdf61d4311bcfca2ac8bc191ea